imports/client/components/RelativeTime.tsx

```diff
@@ -0,0 +1,29 @@
+import { Meteor } from 'meteor/meteor';
+import React, { useEffect, useState } from 'react';
+import { complete, RelativeTimeFormatOpts } from '../../lib/relativeTimeFormat';
+
+const RelativeTime = ({
+  date, minimumUnit, maxElements, terse, now,
+}: {
+  date: Date,
+} & RelativeTimeFormatOpts) => {
+  const [formatted, setFormatted] = useState(complete(date, {
+    minimumUnit, maxElements, terse, now,
+  }));
+
+  useEffect(() => {
+    const timeout = Meteor.setTimeout(() => {
+      setFormatted(complete(date, {
+        minimumUnit, maxElements, terse, now,
+      }));
+    }, formatted.millisUntilChange);
+
+    return () => {
+      Meteor.clearTimeout(timeout);
+    };
+  }, [date, formatted.millisUntilChange, maxElements, minimumUnit, now, terse]);
```

Comment:
I think this might subtly miss updates after the first two, since `millisUntilChange` could plausibly be the same from one update-on-the-minute to the next, and then we'd lose all updates.

I think the most straightforward way to fix that is to have this deps list also include `formatted.formatted`.

Reply:
Good catch, fixed.